.taskcluster.yml

```diff
@@ -97,28 +97,31 @@ tasks:
       dependencies:
         - {$eval: as_slugid("code_checks")}
       provisionerId: proj-relman
-      workerType: ci
+      workerType: generic-worker-ubuntu-22-04
       created: {$fromNow: ''}
       deadline: {$fromNow: '1 hour'}
       payload:
-        features:
-          dind: true
         maxRunTime: 3600
-        image: python:3.10
         env:
           IMAGE: mozilla/taskboot
           REGISTRY: registry.hub.docker.com
           VERSION: "${tag}"
         command:
-          - sh
-          - -lxce
-          - "git clone --quiet ${repository} /src && cd /src && git checkout ${head_rev} -b taskboot &&
-            pip install --no-cache-dir --quiet . &&
-            taskboot --target=/src build --build-tool=dind --image=$IMAGE --tag=$VERSION --write /image.tar tests/dockerfile.empty"
+          - - sh
+            - '-lxce'
+            - |
+              wget https://bootstrap.pypa.io/get-pip.py
+              python3 get-pip.py --user
+              export PATH=$${HOME}/.local/bin:$${PATH}
+              git clone --quiet ${repository} src
+              cd src
+              git checkout ${head_rev} -b taskboot
+              python3 -m pip install --no-cache-dir --quiet --user .
+              taskboot --target=. build --build-tool=podman --image=$${IMAGE} --tag=$${VERSION} --write ../image.tar tests/dockerfile.empty
         artifacts:
-          public/taskboot/test-dind.tar.zst:
+          - name: public/taskboot/test-dind.tar.zst
             expires: {$fromNow: '2 weeks'}
-            path: /image.tar.zst
+            path: image.tar.zst
             type: file
       metadata:
         name: TaskBoot docker build using Docker in Docker
```

Comment:
Nit: could you rename `taskId` and `metadata`->`name` for this and the following task, mentioning podman instead of dind?

Reply:
Ah indeed, good spot!